zad3/app/controllers/bestiary_controller.rb

```diff
@@ -2,11 +2,11 @@
 
 class BestiaryController < ApplicationController
   def index
-
+    @monsters = monsters
   end
 
   def novigrad_monsters
-
+    @monsters = monsters.select { |m| m[:location] == "Novigrad" }
```

Comment:
Mały szczególik - każdą zmienną staramy się nazywać opisowo, nawet jeśli to zajmie kilka znaków więcej 😉